_source/_posts/2023-06-08-identity-developer-podcast.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Unlock identity expertise with our new developer podcast!"
+author: edunham
+by: advocate
+communities: [devops]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/identity-developer-podcast/podcast.jpg 
+type: awareness|conversion
+---
+
+Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 
+
+I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 
+
+The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.
+
+In March, with product manager Megha Rastogi, we discussed what Workforce Identity means, and review some of the history of security that gotu s here. We dig into phishing resistant MFA, and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 
+
+{% youtube <PiY5HDp0ABI> %} 
+
+In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 
+
+{% youtube <oHvukPsaAko> %}
+
+In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 
```

Comment:
Is the branding supposed to be "Devday23" or is "Devday 2023" appropriate?

Reply:
I had no clue so I did it exactly as Auth0 did on the youtube playlist name for the videos